modules/oadp-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= Backing up data from one cluster and restoring it to another cluster
+
+In general, you back up data from one {product-title} cluster and restore it on another {product-title} cluster in the same way you back up and restore data to the same cluster. However, there are some additional prerequisites and differences in procedure when you back up data from one {product-title} cluster and restore it on another.
+
+.Prerequisites
+
+* All relevant prerequisites for backing up and restoring on your platform (AWS, Nicrosoft Azure, etc.) described in the relevant section of this guide, especially for the Data Protection Application (DPA).
```

Comment:
```suggestion
* All relevant prerequisites for backing up and restoring on your platform (for example, AWS, Microsoft Azure, GCP, and so on) are described in the relevant sections of this guide, especially for the Data Protection Application (DPA).
```

I'm a little confused about this part: especially for the Data Protection Application (DPA)

Could we maybe say...

```suggestion
* All relevant prerequisites for backing up and restoring on your platform (for example, AWS, Microsoft Azure, GCP, and so on) are described in the relevant sections of this guide. Additional information is also available for the Data Protection Application (DPA).
```

and keep the same meaning?

Reply:
Note quite -- i hope my rewrite makes e sentence clearer.